public/js/app/components/Table.jsx

```diff
@@ -1,28 +1,33 @@
 import React from 'react'
 
-const renderRow = (row) => (
-  <tr key={row[0]}>
-    {row.map((cell, index) => (
-      <td key={index}>{typeof cell === 'function' ? cell() : cell}</td>
-    ))}
-  </tr>
-)
+const renderRow = (row, rowIndex) => {
+  return (
+    //TODO: unique ID for key
```

Comment:
Do you have an idea about what would be a good ID? This is on cell-level so it feels more difficult to find an ID.